pallets/gear/src/benchmarking/mod.rs

```diff
@@ -179,8 +179,9 @@ fn default_processor_context<T: Config>() -> ProcessorContext {
         message_context: MessageContext::new(
```

Comment:
We could derive `Default` for `MessageContext` and use `Default::default` instead of `MessageContext::new(...)`.

In general, it is good practice to derive `D`-`D`-`C` (`Debug`, `Default`, `Clone`) to all public structs.

Reply:
actually `MessageContext` is not public because truly it uses only inside `core-processor`

Reply:
so, it should be in core-processor but we define it in gear-core I don't know why